redbot/cogs/modlog/modlog.py

```diff
@@ -99,15 +99,22 @@ def __init__(self, bot: Red):
 
     @commands.command()
     @commands.guild_only()
-    async def reason(self, ctx: commands.Context, case: int, *, reason: str = ""):
+    async def reason(self, ctx: commands.Context, *, reason: str):
         """Lets you specify a reason for mod-log's cases
+        
         Please note that you can only edit cases you are
-        the owner of unless you are a mod/admin or the server owner"""
+        the owner of unless you are a mod/admin or the server owner.
+        
+        If no number is specified, the latest case will be used."""
```

Comment:
Since we've modified the command signature, the command syntax will show in help as `[p]reason <reason>`.

You can actually specify a custom signature with the [usage](https://discordpy.readthedocs.io/en/rewrite/ext/commands/api.html#discord.ext.commands.Command.usage) argument to `@commands.command()`. I think this is the best way to make it clear how the command can be used :)

Reply:
So the decorator would look like this:
`[p]reason [case] <reason>`

Reply:
For the arguments, yes, that looks good. You can omit the command name. Play around with it :)

Reply:
Didn't know that was a thing, thanks for the tip!